packages/babel-core/src/transformation/normalize-file.js

```diff
@@ -102,7 +103,8 @@ function parser(pluginPasses, options, code) {
             },
           },
           options,
-        );
+        ) +
+        `\n${generateHelpMessage(err.missingPlugin)}`;
```

Comment:
Can `\n` be added only if `generateHelpMessage` doesn't return `""`?

Reply:
Also can we rename `generateHelpMessage` to `generateMissingPluginMessage`?

Reply:
I'll incorporate both these changes